erc7824-docs/docs/guides/migration-guide.md

```diff
@@ -12,6 +12,419 @@ import TabItem from '@theme/TabItem';
 
 If you are coming from an earlier version of Nitrolite, you will need to account for the following breaking changes.
 
+## 0.5.x Breaking changes
+
+The 0.5.x release includes fundamental protocol changes affecting session keys, channel operations, state signatures, and channel resize rules. The main objective of these changes is to enhance security, and provide better experience for developers and users by ability to limit allowances for specific applications.
+
+**Not ready to migrate?** Unfortunately, at this time Yellow Network does not provide ClearNodes running the previous version of the protocol, so you will need to migrate to the latest version to continue using the Network.
+
+### Protocol Changes
+
+These protocol-level changes affect all implementations and integrations with the Yellow Network.
+
+#### Session Keys: Applications, Allowances, and Expiration
```

Comment:
We should also specify that with the update, all session keys will be deleted and recreated with a new authentication, thus enforcing the expiration.
@philanton @dimast-x , WDYT?